src/dcc/RailcomBroadcastDecoder.hxx

```diff
@@ -86,9 +86,15 @@ private:
     /// bytes arethere to decode. @return dunno.
     bool process_data(const uint8_t *data, unsigned size);
 
+    /// Notifies the state machine that there is no occupancy detected.
+    void notify_empty();
+    
     /// How many times we shall get the same data out of railcom before we
     /// believe it and report to the bus.
-    static const uint8_t REPEAT_COUNT = 3;
+    static const uint8_t MIN_REPEAT_COUNT = 6;
+    /// Maximum number of repeats we remember. This is how many empty packets
+    /// we need to forget the current address when we're getting empty packets.
+    static const uint8_t MAX_REPEAT_COUNT = 8;
```

Comment:
Can you enhance this comment somehow to indicate the intent to count up at 2x the rate that we count down?

Reply:
i changed the names and meaning instead to be more intuitive and not have double counts.